tn_api_hotel.yml

```diff
@@ -239,89 +312,581 @@ components:
+                        room_description:
+                            type: string
+                            description: A description of the room.
+                            example: 'Executive Suite Room'
+                        rate_descriotion:
+                            type: string
+                            description: A description of the rate.
+                            example: '2 Double Bed Non Smoking Featuring All The Modern-day Essentials Including A Flat Screen'
+                        refundable:
+                            type: boolean
+                            description: Explains whether the booking will be refundable.
+                            example: True
+                        smoking:
+                            type: boolean
+                            description: Indicates whether you are allowed to smoke in the room.
+                            example: False
+                        meals:
+                            type: object
+                            properties:
+                                breakfast:
+                                    type: boolean
+                                    description: Indicates if breakfast is provided with the room.
+                                    example: True
+                                lunch:
+                                    type: boolean
+                                    description: Indicates if lunch is provided with the room.
+                                    example: True
+                                dinner:
+                                    type: boolean
+                                    description: Indicates if dinner is provided with the room.
+                                    example: True
+                        bed_type:
+                            type: string
+                            description: The bed that is provided with the room.
+                            example: 'King Single'
 
-    HotelGetRoomsRequest:
-      required:
-        - super_trip_id
-      properties:
-          super_trip_id:
-              type: string
-              example: "bdf876sf6sfsdf87sdf"
-              description: id of the super trip
-          hotel_id:
-            type: string
-            example: 'ndsf6sjf6sdf7sdf6'
-            description: An identifier to refer to the hotel for searching room availability
 
     HotelPriceConfirmRequest:
       required:
-        - super_trip_id
+          - hotels_to_price
       properties:
         super_trip_id:
-          type: string
-          example: "bdf876sf6sfsdf87sdf"
-          description: id of the super trip
-        hotel_id:
-          type: string
-          example: 'ndsf6sjf6sdf7sdf6'
-          description: An identifier to refer to the hotel for searching room availability
+            type: string
+            example: "bdf876sf6sfsdf87sdf"
+            description: Id of the super trip.
+        hotels_to_price:
```

Comment:
Just to confirm as to why you would price confirm multiple hotels at once, when is this request made in the overall flow?

Reply:
We still need to design this part - can we approve these docs and I will make the appropriate changes when we get to pricing in more detail??